thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/rootcause/Aggregator.java

```diff
@@ -0,0 +1,9 @@
+package com.linkedin.thirdeye.rootcause;
+
+import java.util.List;
+import java.util.Map;
+
+
+public interface Aggregator {
```

Comment:
what is the purpose of Aggregator, can we add javadocs to all the interface and their methods